Makefile

```diff
@@ -22,6 +22,6 @@ requirements:
 	poetry export -f requirements.txt --output requirements.txt --without-hashes
 
 proto:
-	 python3 -m grpc_tools.protoc -I=pynumaflow/function/proto --python_out=pynumaflow/function/proto --grpc_python_out=pynumaflow/function/proto  pynumaflow/function/proto/*.proto
-	 python3 -m grpc_tools.protoc -I=pynumaflow/sink/proto --python_out=pynumaflow/sink/proto --grpc_python_out=pynumaflow/sink/proto  pynumaflow/sink/proto/*.proto
-	 sed -i '' 's/^\(import.*_pb2\)/from . \1/' pynumaflow/*/proto/*.py
+	python3 -m grpc_tools.protoc -I=pynumaflow/function/proto --python_out=pynumaflow/function/proto --grpc_python_out=pynumaflow/function/proto  pynumaflow/function/proto/*.proto;\
+   	python3 -m grpc_tools.protoc -I=pynumaflow/sink/proto --python_out=pynumaflow/sink/proto --grpc_python_out=pynumaflow/sink/proto  pynumaflow/sink/proto/*.proto;\
```

Comment:
Is this going to work? the format of the indent is incorrect.

Reply:
It works but let me try to make the format right...